crates/typst/src/foundations/func.rs

```diff
@@ -102,6 +102,12 @@ pub use typst_macros::func;
 /// ]
 /// ```
 ///
+/// # Importing functions
+/// Functions can be imported from one file (module) into another using `#import`.
+/// For example, assume that we have defined the `alert` function from the
+/// previous example in a file called `foo.typ`. We can import it into another
+/// file by writing `#import "foo.typ": alert`.
```

Comment:
I've tried to do this via things like 

    ```ignore
    #import "foo.typ": alert
    ```

But wasn't able to. It failed the tests with "unsupported highlighting language: ignore". Also I wasn't able to preview the output locally (as far as I could tell from https://github.com/typst/typst/discussions/3457#discussioncomment-8515912), so I'm not sure whether it will now highlight.

Can you finish this PR or give me a tip on how I can finish it? 🙂